src/main/java/command/BadCommand.java

```diff
@@ -0,0 +1,26 @@
+package command;
+
+import Dictionary.WordBank;
+import storage.Storage;
+import ui.Ui;
+
+/**
```

Comment:
Add such headers for each different command subclass. Also, if the execute() is nontrivial (long code), do add a header explain what it does and how